hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneClusterImpl.java

```diff
@@ -494,6 +494,9 @@ void initializeConfiguration() throws IOException {
           streamBufferMaxSize.get(), streamBufferSizeUnit.get());
       conf.setStorageSize(OzoneConfigKeys.OZONE_SCM_BLOCK_SIZE, blockSize.get(),
           streamBufferSizeUnit.get());
+      // MiniOzoneCluster should have global pipeline upper limit.
+      conf.setInt(ScmConfigKeys.OZONE_SCM_PIPELINE_NUMBER_LIMIT,
+          pipelineNumLimit == 3 ? 2 * numOfDatanodes : pipelineNumLimit);
```

Comment:
Can you add more comments here? If the # of SCM pipeline limit is 3, should we set OZONE_SCM_PIPELINE_NUMBER_LIMIT to 3 instead of 2 * numOfDatanodes?

Reply:
3 means the default number pipelineNumLimit, which means there is no use-set limit. I should've made this a global reference.